docs/diagrams/orchestration/identity/identity.mmd

```diff
@@ -0,0 +1,49 @@
+sequenceDiagram
+    autonumber
+    participant rp as Relying Party
+    participant orch_api as Orchestration OIDC API
+    participant auth as Authentication
+    participant ipv as IPV Core
+    participant spot as SPOT
+    Note over rp,orch_api: Dotted lines indicate 302 redirects of the user's browser
+    rp -->> orch_api : OIDC Authentication request to /authorize
+    orch_api  ->> orch_api : Validates Authentication request
+    orch_api ->> orch_api : Create and store session, set cookie
+    orch_api -->> auth : OAuth 2.0 Authorization request to /authorize
+    auth ->> auth : Authenticates the user
+    auth -->> orch_api : Return the user with an auth code query parameter
+    Note over orch_api,auth: Solid lines indicate machine-machine API calls
+    orch_api ->> auth : Token request containing auth code to /token
+    auth ->> orch_api : Token response containing access token
+    orch_api ->> auth : Userinfo request containing access token to /userinfo
+    auth ->> orch_api : Userinfo response
+    orch_api -->> ipv : OAuth 2.0 Authorization request to /authorize
+    ipv -->> orch_api : Return the user with an auth code query parameter
+    orch_api ->> ipv : Exchange the auth code for an access token with a signed client assertion
+    ipv ->> orch_api : Issue an access token
+    orch_api ->> ipv : Request for user identity authorised with the access token
+    ipv ->> orch_api : User identity response
```

Comment:
```suggestion
    ipv ->> orch_api : Useridentity response
```

Reply:
I think I prefer this with the space. It's not a concept in the same way as `userinfo` is